src/index.js

```diff
@@ -9,113 +9,98 @@ export function h(type, props) {
 
   while (stack.length) {
     if (Array.isArray((node = stack.pop()))) {
-      for (i = node.length; i--; ) {
+      for (var i = node.length; i--; ) {
         stack.push(node[i])
       }
-    } else if (null == node || node === true || node === false) {
+    } else if (null == node || true === node || false === node) {
     } else {
-      children.push(typeof node === "number" ? (node = node + "") : node)
+      children.push(typeof node === "number" ? (node += "") : node)
     }
   }
 
-  return typeof type === "string"
+  return typeof tag === "string"
     ? {
-        type: type,
+        tag: tag,
```

Comment:
Why are we naming `type` back to `tag` again?

(I never knew why we changed it to type in the first place...)

Reply:
I noticed this.. and @selfup noted that it might be easier for everyone if we don't bundle these changes in with this proposal. Perhaps @JorgeBucaran could revert these bits and make a separate PR with `type -> tag` if it is something that we want to do. It would make for a cleaner diff here.

Reply:
👍